phd-tests/framework/src/test_vm/mod.rs

```diff
@@ -926,7 +926,6 @@ impl TestVm {
     // unfortunately I don't know how to plumb the futures for that, since we'd
     // have to close over `&self`, so doing any Boxing to hold an
     // `async move {}` immediately causes issues.
-    #[must_use]
```

Comment:
yeah, the `Result` makes this already `must_use`...but perhaps there should be a type-level `#[must_use]` on the `ShellOutput` struct itself?

Reply:
`ShellOutput` is now fully gone :saluting_face: